atom/browser/common_web_contents_delegate_mac.mm

```diff
@@ -27,15 +28,17 @@ - (void)redispatchKeyEvent:(NSEvent*)event;
   if (event.windows_key_code == ui::VKEY_ESCAPE && is_html_fullscreen())
     ExitFullscreenModeForTab(source);
 
-  if (!ignore_menu_shortcuts_) {
-    // Send the event to the menu before sending it to the window
-    if (event.os_event.type == NSKeyDown &&
-        [[NSApp mainMenu] performKeyEquivalent:event.os_event])
-      return;
-
-    if (event.os_event.window &&
-        [event.os_event.window isKindOfClass:[EventDispatchingWindow class]])
-      [event.os_event.window redispatchKeyEvent:event.os_event];
+  if (auto* web_preferences = WebContentsPreferences::From(source)) {
+    if (!web_preferences->IsEnabled("ignoreMenuShortcuts", false)) {
```

Comment:
The default value of `IsEnabled` is `false`, incase a shorthand version looks better here.